src/views/clusteroverview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/VirtualizationFeaturesWizard/components/SummaryStep/components/HighAvailabilitySection/HighAvailabilitySummarySection.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from 'react';
+
+import ExpandSectionWithCustomToggle from '@kubevirt-utils/components/ExpandSectionWithCustomToggle/ExpandSectionWithCustomToggle';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import {
+  FENCE_AGENTS_OPERATOR_NAME,
+  NODE_HEALTH_OPERATOR_NAME,
+} from '@overview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/utils/constants';
+import { OperatorDetailsMap } from '@overview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/utils/hooks/useVirtualizationOperators/utils/types';
+import FeatureSummaryItem from '@overview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/VirtualizationFeaturesWizard/components/SummaryStep/components/FeatureSummaryItem/FeatureSummaryItem';
+import { getHighAvailabilityInstallState } from '@overview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/VirtualizationFeaturesWizard/components/SummaryStep/components/HighAvailabilitySection/utils/utils';
+import SummaryStatusIcon from '@overview/SettingsTab/ClusterTab/components/VirtualizationFeaturesSection/VirtualizationFeaturesWizard/components/SummaryStep/components/SummaryStatusIcon/SummaryStatusIcon';
+import { Stack, StackItem } from '@patternfly/react-core';
+
+import './HighAvailabilitySummarySection.scss';
+
+type HighAvailabilitySummarySectionProps = {
+  operatorDetailsMap: OperatorDetailsMap;
+};
+
+const HighAvailabilitySummarySection: FC<HighAvailabilitySummarySectionProps> = ({
+  operatorDetailsMap,
+}) => {
+  const { t } = useKubevirtTranslation();
+
+  const { installState: nhcInstallState } = operatorDetailsMap?.[NODE_HEALTH_OPERATOR_NAME];
+  const { installState: farInstallState } = operatorDetailsMap?.[FENCE_AGENTS_OPERATOR_NAME];
```

Comment:
```suggestion
  const { installState: nhcInstallState } = operatorDetailsMap?.[NODE_HEALTH_OPERATOR_NAME] ?? {};
  const { installState: farInstallState } = operatorDetailsMap?.[FENCE_AGENTS_OPERATOR_NAME] ?? {};
```

This will raise and error if operatorDetailsMap?.FENCE_AGENTS_OPERATOR_NAME is undefiend